### PR TITLE
Plugin data bootstrapping and module namespacing

### PR DESCRIPTION
**Pull request by rtibbles**
### Summary
* Refactors bootstrapping template tags that bring Python code into the frontend to use a standard `plugin_data` API.
* Makes various dependency updates for polyfills.
* Namespaces frontend module bundles by the kolibri plugins they are a part of

### Reviewer guidance
Does everything still work as intended? Is the code clear and comprehensible?

https://github.com/learningequality/kolibri-exercise-perseus-plugin/compare/v1.2.1...v1.2.2a1

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [x] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/plugins/device_management/assets/src/views/ManageContentPage/ChannelListItem.vue [hunk 1]
```diff
@@ -87,7 +87,7 @@
 
   import { mapGetters } from 'vuex';
   import CoachContentLabel from 'kolibri.coreVue.components.CoachContentLabel';
-  import KResponsiveWindowMixin from 'kolibri-components/src/KResponsiveWindowMixin';
+  import responsiveWindowMixin from 'kolibri.coreVue.mixins.responsiveWindowMixin';
```
Comment (nucleogenesis) [inline kolibri/plugins/device_management/assets/src/views/ManageContentPage/ChannelListItem.vue:90]: Not sure if this is just a naming change, but there is some padding or margin missing somewhere now.

missing-margin

Comment (nucleogenesis) [reply kolibri/plugins/device_management/assets/src/views/ManageContentPage/ChannelListItem.vue:90]: Elsewhere, again.

missing-margin-part-duex

**Review by nucleogenesis — APPROVED** [review]
Looks good - the issues I noted earlier are actually issues on develop, not due to changes here.

File: kolibri/plugins/device_management/assets/src/views/ManageContentPage/ChannelListItem.vue (hunk 1 above)
Comment (nucleogenesis) [reply kolibri/plugins/device_management/assets/src/views/ManageContentPage/ChannelListItem.vue:90]: Ah - the above are already an issue on develop https://github.com/learningequality/kolibri/issues/5868

**Comment by nucleogenesis** [comment]
@rtibbles Just a note about https://github.com/learningequality/kolibri/pull/5796 - I don't think much will change in terms of how everything works but I think I'll need to change a couple lines of code once this is merged and I rebase on develop.

File: kolibri/plugins/device_management/assets/src/views/ManageContentPage/ChannelListItem.vue (hunk 1 above)
Comment (rtibbles) [reply kolibri/plugins/device_management/assets/src/views/ManageContentPage/ChannelListItem.vue:90]: Phew - I didn't know how I could have caused that :)

**Pull request merged**